### DISCUSSION TITLE
Install to the emmc

### DISCUSSION THREAD

**Discussion by afgp87** [devmfc/debian-on-amlogic#99] [Q&A]
First, i want to thank @devmfc for all the work you are doing and sharing it with us. 
I need help. I need to install debian in the emmc I try to run ./aml-install-to-emmc I don't want to keep the old android firmware. I just want to have debian Install the multiboot and run the command again. It said that work, but I removed the sd card and tried to boot from the emmc, but I can't ssh

**Comment by afgp87** [comment]
> First, i want to thank @devmfc for all the work you are doing and sharing it with us. 
> I need help. I need to install debian in the emmc I try to run ./aml-install-to-emmc I don't want to keep the old android firmware. I just want to have debian Install the multiboot and run the command again. It said that work, but I removed the sd card and tried to boot from the emmc, but I can't ssh

I redid everything again, and it works. I installed the multiboot and then installed it to the emmc, removed the power and sd card, and power again it boot to debian

Now my klipper server is up and running thank you ![Screenshot_20240927_124520_Samsung Internet.jpg](https://github.com/user-attachments/assets/ce97cfbb-398b-46b6-a540-3d153a403258)

**Reply by devmfc** [reply]
Great to hear! You're welcome!
